bin/node/runtime/src/lib.rs

```diff
@@ -356,7 +356,10 @@ impl pallet_scheduler::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 	type MaximumWeight = MaximumSchedulerWeight;
 	type ScheduleOrigin = EnsureRoot<AccountId>;
+	#[cfg(feature = "runtime-benchmarks")]
 	type MaxScheduledPerBlock = ConstU32<512>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type MaxScheduledPerBlock = ConstU32<50>;
```

Comment:
You could also give the pallets `referenda` and `conviction-voting` a:  
```rust
#[cfg(feature = "runtime-benchmarks")]
type MaxScheduledPerBlock = ConstU32<512>;
```
or?

Reply:
Not sure I understand
Check another solution https://github.com/paritytech/substrate/pull/13143

Reply:
If pallets referenda and conviction-voting has something like MaxScheduledPerBlock, they assume there is a limit per block at scheduler. Where in fact they don't know about scheduler internals, they only require some scheduler trait to be implemented.

Reply:
@ggwpez what do you think?
I think with the changes in this PR we make ti easier to debug or prevent the issue without adding any additional complexity

Reply:
Yes thinking about this again the easy approach is good.  
Just need to check that the Polkadot benches still work, since I think the companion system does not check that.

Reply:
Do you mean to check if the changes in this PR break anything in dependent repos?
I think there is no such changes